pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -19,25 +33,49 @@
     <arg name="publish_tf" value="$(arg publish_objects_tf)" />
   </include>
 
-  <node name="pointcloud_screenpoint_nodelet" pkg="nodelet" type="nodelet"
-        args="load jsk_pcl/PointcloudScreenpoint $(arg manager)"
-        output="screen" clear_params="true" respawn="true"
-        machine="$(arg machine)">
-    <remap from="~points" to="/kinect_head/depth_registered/throttled/points" />
-    <remap from="~point" to="/kinect_head/rgb/throttled/image_rect_color/screenpoint" />
+  <group if="$(arg remote)">
+    <node name="pointcloud_screenpoint_nodelet" pkg="nodelet" type="nodelet"
+          args="load jsk_pcl/PointcloudScreenpoint $(arg manager)"
+          output="screen" clear_params="true" respawn="true"
+          machine="$(arg machine)">
+      <remap from="~points" to="$(arg POINT_CLOUD)" unless="$(arg remote)" />
+      <remap from="~points" to="$(arg POINT_CLOUD_REMOTE)" if="$(arg remote)"/>
+      <remap from="~point" to="$(arg RGB_IMAGE)/screenpoint" unless="$(arg remote)" />
+      <remap from="~point" to="$(arg RGB_IMAGE_REMOTE)/screenpoint" if="$(arg remote)" />
+      <rosparam>
+        queue_size: 10
+        crop_size: 10
+        search_size: 16
+      </rosparam>
+    </node>
+  </group>
+
+  <node name="prosilica_decompress" pkg="image_transport" type="republish"
+        args="compressed in:=$(arg PROSILICA_IMAGE) out:=$(arg PROSILICA_IMAGE_REMOTE)" />
+  <node name="rgb_decompress" pkg="image_transport" type="republish"
+        args="compressed in:=$(arg RGB_IMAGE) out:=$(arg RGB_IMAGE_REMOTE)" />
```

Comment:
in and out should be in remap tag
args="compressedDepth raw"

example:
https://github.com/jsk-ros-pkg/jsk_recognition/blob/b5657ed28937616d2abf3d3f967c69ccf7a85f28/jsk_pcl_ros/launch/openni2_remote.launch#L80-L85

http://wiki.ros.org/image_transport

Reply:
Fix by 78880de9ec9c6cfab05932ac12e99239e7fb9373